projects/ng-swiper/src/lib/swiper/swiper.component.spec.ts

```
@@ -0,0 +1,79 @@
+        <div ngSwiperItem>Item 6</div>
+        <div ngSwiperItem>Item 7</div>
+        <div ngSwiperItem>Item 8</div>
+    </ng-swiper>`,
+})
+class TestSwiperComponent extends AbstractTestSwiperComponent {
+    options = input<SwiperOptions>(NG_SWIPER_DEFAULT_OPTIONS);
+}
+
+describe('SwiperComponent', () => {
+    let component: AbstractTestSwiperComponent;
+    let swiper: SwiperComponent;
```

ESLint

[error] 'swiper' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)